### PR TITLE
mention path in rladmin command

### PR DESCRIPTION
**Pull request by vikrammoule-redislabs**
without mentioning the absolute path the command results in an error `ERROR: invalid token 'saslauthd.conf'`

### CODE REVIEW

File: content/platforms/kubernetes/tasks/ldap-on-k8s.md [hunk 1]
```diff
@@ -159,10 +159,10 @@ ldap_password: admin
 
 ### Using rladmin to update the configuration
 
-If this information was in a file called “ldap.conf”, you can connect to a Redis Enterprise Node pod and configure the server via:
+If this information was in a file called “ldap.conf” stored in "/tmp", you can connect to a Redis Enterprise Node pod and configure the server via:
 
 ```sh
-rladmin cluster config saslauthd_ldap_conf ldap.conf
+rladmin cluster config saslauthd_ldap_conf /tmp/ldap.conf
```
Comment (kaitlynmichaelredislabs) [inline content/platforms/kubernetes/tasks/ldap-on-k8s.md:165]: We just merged a PR from @laurentdroin editing this file to `/path/to/ldap.conf`. Should it be specifically mentioned that this is stored in `/tmp` instead?

**Review by kaitlynmichaelredislabs — COMMENTED** [review]
This PR edits the same lines as #1258 , we need to resolve any conflicts or close this as a duplicate.

File: content/platforms/kubernetes/tasks/ldap-on-k8s.md (hunk 1 above)
Comment (laurentdroin) [reply content/platforms/kubernetes/tasks/ldap-on-k8s.md:165]: No, I don't think there is any assumption that the file would be in /tmp.
The lines above say:

```
If this information was in a file called “ldap.conf”, you can connect to a Redis Enterprise Node pod and configure the server via:

rladmin cluster config saslauthd_ldap_conf ldap.conf
```

So, no assumption where the file is (IMO).

**Review by kaitlynmichaelredislabs — APPROVED** [review]
Approved via discussion in slack #redislabs-docs

**Pull request merged**